sdk/tests/conformance2/extensions/ext-texture-norm16.html

```diff
@@ -153,6 +153,23 @@
                                 "framebuffer should not be complete with SNORM16 texture attached");
 }
 
+function runInternalFormatQueryTest()
+{
+    debug("");
+    debug("testing the internal format query");
+
+    var maxSamples = gl.getParameter(gl.MAX_SAMPLES);
+    var formats = new Array(ext.R16_EXT, ext.RG16_EXT, ext.RGBA16_EXT);
+    for (var fmt = 0; fmt < formats.length; ++fmt) {
```

Comment:
`for (const format of formats) {`

Reply:
only fixed this for loop style. The copy src is using idx for some logic so left unchanged